### DISCUSSION TITLE
Bulk update API

### DISCUSSION THREAD

**Discussion by churiart** [phalcon/cphalcon#15268] [Q&A]
Do we have a bulk update API in phalcon using resources IDs or I should write a raw query for it? E.g.

```
User::query()->inWhere('id', [1,2,3])
    ->update(['status' => 1]);
```

**Comment by Jeckerson** [comment]
```
User::find([
        'id IN ({ids:array})',
        'bind' => ['ids' => [1, 2, 3]],
    ])
    ->update(['status' => 1]);
```

**Important note:** Max quantity of bind parameters (in this case ids) is `65535`. So do not pass array of more then `65535` values. This is MySQL limitation (in case you use this DB).
